dev/integration_tests/ios_app_with_extensions/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -907,6 +915,25 @@
 			defaultConfigurationName = Release;
 		};
 /* End XCConfigurationList section */
+
+/* Begin XCRemoteSwiftPackageReference section */
+		F735C5C22485E50A002BE868 /* XCRemoteSwiftPackageReference "example-package-playingcard" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/apple/example-package-playingcard.git";
```

Comment:
This was the most basic Apple Swift Package I could find.  When we run the integration test it will check out this dependency from the GitHub repo.

Reply:
This is the example package used in the [Swift docs](https://swift.org/package-manager/#example-usage) so hopefully if they every remove it they will replace it with some other example we can easily migrate.  Fortunately it's not actually being used anywhere, so it will be extremely trivial to replace with some different package.